app/src/main/java/org/dhis2/utils/DateUtils.java

```diff
@@ -694,38 +701,37 @@ public Date getNextPeriod(PeriodType period, Date currentDate, int page, boolean
     }
 
     public String getPeriodUIString(PeriodType periodType, Date date, Locale locale) {
```

Comment:
Is there an specific reason why we are leaving this method in the DateUtils class if it was just implemented in the DhisPeriodUtils class??

Reply:
Forgot to remove it. Done!